docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md

```diff
@@ -0,0 +1,100 @@
+---
+sidebar_label: 'Manage Data Assets'
+title: 'Manage Data Assets'
+description: Create and manage Data Assets in GX Cloud.
+---
+
+A Data Asset is a collection of records that you create when you connect to your Data Source. When you connect to your Data Source, you define a minimum of one Data Asset. You use these Data Assets to create the Batch Requests that define the data that is provided to your Expectations.
```

Comment:
Batch Requests specify a subset of data to select, as opposed to defining the data itself, so I would change `that define the data` to something like `to select the data`.